GTFO-API/Utilities/LiveEdit.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GTFO.API.Utilities
+{
+    /// <summary>
+    /// LiveEditEventHandler
+    /// </summary>
+    /// <param name="e">EventArgs</param>
+    public delegate void LiveEditEventHandler(FileSystemEventArgs e);
+
+    /// <summary>
+    /// Utility class to make support of LiveEdit config file for plugins
+    /// </summary>
+    public static class LiveEdit
+    {
+        private static readonly List<LiveEditListener> m_Listeners = new ();
+
+        /// <summary>
+        /// Create the LiveEdit Listener and Allocate
+        /// </summary>
+        /// <param name="path">Base Path to Track the change</param>
+        /// <param name="filter">File filter to filter the files ie) *.json</param>
+        /// <param name="includeSubDir">Include Sub-directories?</param>
+        public static LiveEditListener CreateListener(string path, string filter, bool includeSubDir)
+        {
+            var listener = new LiveEditListener(path, filter, includeSubDir);
```

Comment:
Use explicit declaration
```suggestion
            LiveEditListener listener = new(path, filter, includeSubDir);
```